frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/outline/outline_block_component.dart

```diff
@@ -0,0 +1,233 @@
+import 'dart:async';
+
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+class OutlineBlockKeys {
+  const OutlineBlockKeys._();
+
+  static const String type = 'outline_block';
+}
+
+// defining the callout block menu item for selection
+SelectionMenuItem outlineItem = SelectionMenuItem.node(
+  name: 'Outline',
+  iconData: Icons.clear_all,
```

Comment:
clear_all?

Reply:
I have selected a few icons which match our purpose here, let me know which feels the best selection here:

![icons_selection](https://github.com/AppFlowy-IO/AppFlowy/assets/37607224/9a58e8f4-3d12-4b5c-89d2-1a9acd593d4d)

Reply:
The list_alt looks better.